client/test/TestFluid.res

```diff
@@ -683,7 +683,7 @@ let run = () => {
     t(
       "insert into middle string",
       mlStr,
-      ~pos= /* quote + 2 + newline */44,
+      ~pos=/* quote + 2 + newline */ 44,
```

Comment:
Were these manual formatting changes, or are you somehow auto-formatting .res files?
(I ask because `./scripts/formatter/format format` doesn't seem to change the files for me.)

Reply:
I guess I saved these files with my editor, and the rescript plugin acts differently from whatever the script calls?

Something to fix after we move to v9 I think.